barcode/barcode_utils.py

```diff
@@ -2,8 +2,8 @@
 
 import base64
 import struct
-import skip32
-import code128
+import barcode.skip32
+import barcode.code128
 import os
```

Comment:
I had to move some stuff into the barcode modules, so these were all prefixed with barcode.  I know there's an easier way to do this here, just didn't have time.